src/main/java/tr/com/srdc/cda2fhir/transform/ResourceTransformerImpl.java

```diff
@@ -897,6 +903,29 @@ public EntryResult tClinicalDocument2Bundle(ClinicalDocument cdaClinicalDocument
 			}
 		}
 
+		// documentationOf.serviceEvent.assignedEntity -> attester[mode = professional]
```

Comment:
So I don't think we want to make a service event assigned entity an attester. Quoting the spec, "The serviceEvent/performer represents the healthcare providers involved in the current or pertinent historical care of the patient. Preferably, the patient’s key healthcare providers would be listed, particularly their primary physician and any active consulting physicians, therapists, and counselors." I'd be more inclined to add it as a detail reference to any of the event itself, but that's my take.

Reply:
But yes, Bill's comments seem to indicate this makes more sense as an attester. However, we already have two attestors supported. I could see this being mapped to either, my personal feeling is that it should be detail. If we just tell them we did it that way I'd think it'd work, but I'd be curious to hear Afsin's opinion.

Reply:
We should ticket out supporting code on the event detail, as something we can evaluate when we look at real data. the spec is a little weird on it (the example differs from the content), and I imagine what we see from Epic/Cerner will vary substantially.

Reply:
Ok. Would we at all indicate that the individual is a primary car provider?

Reply:
Ideally you would with the code element I think.